docs/beamrunner/reference.md

```diff
@@ -71,13 +71,33 @@ The Streams Runner package contains the following directories:
     </tr>
     <tr>
       <td><code class="highlighter-rouge">tracingLevel</code></td>
-      <td>Set the tracing and logging level of StreamsRunner translation and runtime. Levels: <code class="highlighter-rouge">ERROR</code>, <code class="highlighter-rouge">WARN</code>, <code class="highlighter-rouge">INFO</code>, <code class="highlighter-rouge">DEBUG</code>, <code class="highlighter-rouge">TRACE</code></td>
+      <td>Set the tracing and logging level of StreamsRunner translation and runtime. If specified, overrides all other tracing options. Levels: <code class="highlighter-rouge">ERROR</code>, <code class="highlighter-rouge">WARN</code>, <code class="highlighter-rouge">INFO</code>, <code class="highlighter-rouge">DEBUG</code>, <code class="highlighter-rouge">TRACE</code></td>
+      <td><code class="highlighter-rouge">[null]</code></td>
+    </tr>
+    <tr>
+      <td><code class="highlighter-rouge">traceTranslation</code></td>
+      <td>Set the translation trace level for the application. Specify a single level: <code class="highlighter-rouge">ERROR</code>, <code class="highlighter-rouge">WARN</code>, <code class="highlighter-rouge">INFO</code>, <code class="highlighter-rouge">DEBUG</code>, <code class="highlighter-rouge">TRACE</code> to set all loggers to level and/or set individual components. Components include Runner, Streams. Example (set all to <code class="highlighter-rouge">INFO</code> except StreamsRunner logging to <code class="highlighter-rouge">WARN</code>):<br /><br /><code class="highlighter-rouge">INFO,Runner=WARN</code></td>

```

Comment:
"Components include Runner and Streams. For example, to set Streams Runner to WARN and all other loggers to INFO, specify the following:"